### ISSUE TITLE
error

### ISSUE THREAD

**Issue by joaoadao** [ruffle-rs/ruffle#2256]
# Error Info
Error name: TypeError
Error message: Failed to execute 'compile' on 'WebAssembly': HTTP status code is not ok
Error stack:
```
TypeError: Failed to execute 'compile' on 'WebAssembly': HTTP status code is not ok
```

# Player Info
Allows script access: false
Player type: Object
SWF URL: https://img.clickjogos.com.br/dl/b/bc816b4c94df2e95b39d815e453d4389/src.swf
Param movie: https://img.clickjogos.com.br/dl/b/bc816b4c94df2e95b39d815e453d4389/src.swf
Param allowScriptAccess: never
Attribute 0: [object Attr]
Attribute 1: [object Attr]
Attribute 2: [object Attr]
Attribute 3: [object Attr]
Attribute 4: [object Attr]

# Page Info
Page URL: https://www.clickjogos.com.br/jogos/viaduct-designer

# Browser Info
Useragent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36
OS: Win32

# Ruffle Info
Ruffle version: 0.1.0
Ruffle source: undefined

**Comment by Futurama39** [comment]
![image](https://user-images.githubusercontent.com/12499845/104365614-8098b680-5518-11eb-9d96-301237bcfa47.png)

Proposing to be closed in favour of #1368

**Comment by Toad06** [comment]
This was an issue on Clickjogos's end - now fixed.
